### PR TITLE
Feature/bal 2 gp login

### PR DESCRIPTION
**Pull request by gperissetcelteeka**
# Login/Create account Feature

## :recycle: Current situation & Problem
*Describe the current situation (if possible, with an exemplary (or real) code snippet and/or where this is used)*
*Please link any open issue that is addressed with this PR*
Related issues links
#8 Change avatar from profile 
#25 Change avatar on sigup
#7 Signup user

## :bulb: Proposed solution
*Describe the solution and how this affects the project and internal structure*

## :gear: Release Notes 
*Add a summary of the feature and possible migration guides if this is a breaking change so this section can be added to the release notes.*
*Include code snippets that provide examples of the feature implemented if it appends or changes the public interface.*

## :heavy_plus_sign: Additional Information
*Provide some additional information if possible*

### Related PRs
*Reference the related PRs*

### Testing
*Are there tests included? If yes, which situations are tested, and which corner cases are missing?*

### Reviewer Nudging
*Where should the reviewer start? Where is a good entry point?*

### Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md):
- [x] I agree to follow the [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: Balance/Home.swift [hunk 1]
```diff
@@ -181,11 +182,11 @@ struct HomeView: View {
...
 
 #if DEBUG
-struct MainView_Previews: PreviewProvider {
+struct Home_Previews: PreviewProvider {
     static var previews: some View {
         HomeView()
-            .environmentObject(BalanceScheduler())
-            .environmentObject(MockDataStorageProvider())
+            .environmentObject(Account(accountServices: []))
+            .environmentObject(FirebaseAccountConfiguration<FHIR>(emulatorSettings: (host: "localhost", port: 9099)))
```
Comment (sebastianricaldoni) [inline Balance/Home.swift:189]: @gperissetcelteeka is there a way we can configure so we don't have to hardcode `localhost` and port?

File: Balance/Login/AuthViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+    //
+    //        let database = Firestore.firestore()
+    //        database.collection("users").document("\(uid)/avatar").setData(["avatar": "test"]) { err in
+    //            if let err = err {
+    //                print("Error writing document: \(err)")
+    //            } else {
+    //                print("Document successfully written!")
+    //            }
+    //        }
+    //    }
```
Comment (sebastianricaldoni) [inline Balance/Login/AuthViewModel.swift:157]: @gperissetcelteeka can we remove this block? 
```suggestion
```

File: Balance/Login/AuthViewModel.swift [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+    //            }
+    //            Auth.auth().currentUser?.reload(completion: { err in
+    //                if let err = err {
+    //                    print(err.localizedDescription)
+    //                }
+    //                print(Auth.auth().currentUser?.photoURL ?? "")
+    //            })
+    //        })
+    //        Auth.auth().currentUser?.reload()
+    //    }
```
Comment (sebastianricaldoni) [inline Balance/Login/AuthViewModel.swift:193]: ditto
```suggestion
```

**Review by sebastianricaldoni — APPROVED** [review]
LGTM! I only left a few minor comments
